src/elements/content-explorer/stories/tests/ContentExplorer-visual.stories.js

```diff
@@ -1,25 +1,152 @@
 import { http, HttpResponse } from 'msw';
+import { expect, userEvent, waitFor, within, screen } from '@storybook/test';
 
-import { userEvent, waitFor, within } from '@storybook/test';
 import ContentExplorer from '../../ContentExplorer';
-import { DEFAULT_HOSTNAME_API } from '../../../../constants';
 import mockRootFolder from '../__mocks__/mockRootFolder';
-import { defaultVisualConfig, SLEEP_TIMEOUT } from '../../../../utils/storybook';
+import mockSubFolder from '../__mocks__/mockSubFolder';
+import { defaultVisualConfig } from '../../../../utils/storybook';
+
+import { DEFAULT_HOSTNAME_API } from '../../../../constants';
+
+export const basic = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
 
-export const basic = {};
+        expect(await canvas.findByText('An Ordered Folder')).toBeInTheDocument();
+        expect(canvas.getByText('Tue Apr 16 2019 by Jeremy Press')).toBeInTheDocument();
+
+        expect(canvas.getByText('Book Sample.pdf')).toBeInTheDocument();
+        expect(canvas.getByText('Thu Dec 8 2022 by Jeremy Press')).toBeInTheDocument();
+    },
+};
 
 export const openExistingFolder = {
     play: async ({ canvasElement }) => {
         const canvas = within(canvasElement);
-        await waitFor(
-            async () => {
-                const row = canvas.getByText('An Ordered Folder');
-                await userEvent.click(row);
-            },
-            {
-                timeout: SLEEP_TIMEOUT,
-            },
-        );
+
+        const subFolder = await canvas.findByText('An Ordered Folder');
+        await userEvent.click(subFolder);
+
+        expect(await canvas.findByText('Audio.mp3')).toBeInTheDocument();
+    },
+};
+
+export const openDeleteConfirmationDialog = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+
+        const moreOptionsButton = await canvas.findByTestId('bce-btn-more-options');
```

Comment:
i think you can try `findByRole('button', { name: 'More options' })`;

same for below